inputs/test-fixtures/datatable-advanced.yaml

```
@@ -0,0 +1,46 @@
+dashboards:
+  - name: "[Test] Datatable - Advanced"
+    description: "Test fixture for datatable visualization with sorting and formatting"
+    panels:
+      # ES|QL variant
+      - type: charts
+        title: "Top Agents by System Load"
+        grid: { x: 0, y: 0, w: 24, h: 15 }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)